modules/atomic-widgets/widgets/atomic-image.php

```diff
@@ -13,28 +13,65 @@
 }
 
 class Atomic_Image extends Atomic_Widget_Base {
-	public function get_icon() {
-		return 'eicon-image';
+	public function get_name() {
+		return 'a-image';
 	}
 
 	public function get_title() {
 		return esc_html__( 'Atomic Image', 'elementor' );
 	}
 
-	public function get_name() {
-		return 'a-image';
+	public function get_icon() {
+		return 'eicon-image';
 	}
 
 	protected function render() {
 		$settings = $this->get_atomic_settings();
 
-		$image_url = $settings['image']['url'];
+		$attrs = array_filter( array_merge(
+			$settings['image'] ?? [],
+			[ 'class' => $settings['classes'] ?? '' ]
+		) );
+
+		Utils::print_wp_kses_extended(
+			sprintf(
+				'<img %1$s >',
```

Comment:
```suggestion
				'<img %1$s />',
```

OR

```suggestion
				'<img %1$s>',
```

Reply:
[self close in ](https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags)
I think it is OK to keep it that way, but lets talk about it if you wish